imports/ui/components/review/review.js

```diff
@@ -62,7 +62,14 @@ Template.review.helpers({
       applicant.scores = [];
       applicant.weight = 0;
       applicant.notes = [];
+      applicant.interviewer_name = [];
       _.each(interviewings, function(i) {
+        const usernameToPush = Meteor.users.findOne({ "emails": { $elemMatch: { "address": i.user_email, "verified":false } } }).username
```

Comment:
Have you tested that the /review UI works? On my side it doesn't list any applicants because it's getting an `undefined` returned from this findOne. This seems to be caused by the client-side not having a subscription to something that publishes part of the users collection -- you'll have to implement this (I didn't realize this either 😮 ). DM me for discussion.